<a href="https://colab.research.google.com/github/LelioWolf/Python_test/blob/main/dir_test/Exercicios.github_ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Exercício 1

In [1]:
print ('mundo,mundo, vasto mundo')

mundo,mundo, vasto mundo


Exercício 2

In [2]:
help ('print')

Help on built-in function print in module builtins:

print(...)
    print(value, ..., sep=' ', end='\n', file=sys.stdout, flush=False)
    
    Prints the values to a stream, or to sys.stdout by default.
    Optional keyword arguments:
    file:  a file-like object (stream); defaults to the current sys.stdout.
    sep:   string inserted between values, default a space.
    end:   string appended after the last value, default a newline.
    flush: whether to forcibly flush the stream.



Exercício 3

Se você fizer uma corrida de 10 quilômetros em 43 minutos e 30 segundos, qual será seu tempo médio por milha? Qual é a sua velocidade média em milhas por hora? (Dica: há 1,61 quilômetros em uma milha).


In [8]:
10 / 1.61

6.211180124223602

In [9]:
43.20 / 6.21

6.9565217391304355

In [10]:
43.20 / 6.95

6.215827338129497

In [13]:
43.20 / 60

0.7200000000000001

In [15]:
6.21 * 0.72

4.4712

Logo, o tempo médio por milha será de 6 minutos e 21 segudos. E a sua velocidade média será de 4.47 milhas/h.